app/metrics.yaml

```diff
@@ -1847,6 +1847,7 @@ search.default_engine:
       - metrics
```

Comment:
For the description above: What value is used in case of a custom search engine? If it's really `"custom"` then that will result in no value but errors being recorded. The URL metric type does validate its input to be a sort-of valid URL (starts with a scheme like `https`, is ascii-only  plus a few extra characters).

Reply:
Yes, I have noticed that in the documentation. According to [this](https://github.com/mozilla-mobile/fenix/issues/1607), we do not want to send the URL of custom engines added by the user. If that is still the case I thought of 3 ways we could handle this :
 - just let it resolve to `invalid_value` and keep this in mind when analyzing data (what the current code does)
 - do not send any metric for custom engines (I am not sure we want this, we would lose related data)
 - send a custom URL agreed upon beforehand. Something like `"https://custom"`
What do you think?

Reply:
@mcarare I think that a "magic" value like `https://custom` would work, just be sure and check with the data-science folks or the stakeholders looking at the data to ensure that whatever tools they are using to analyze this is prepared to handle this correctly. My concern being that downstream tools that are consuming this may need updates in order to properly display this.

Reply:
Is it not possible to send a null value? It would seem strange that you wouldn't be allowed to send an empty url.